behavioral_analytics/track.py

```diff
@@ -0,0 +1,193 @@
+import json
+import logging
+import math
+import numbers
+import operator
+import re
+
+from esrally.track.params import *
+
+
+class EventBatchDataReader:
```

Comment:
Can you please add a comment mentioning that most of this was copy/pasted from Rally?